serve_index.py

```diff
@@ -0,0 +1,160 @@
+import sys
+import os
+import time
+import statistics
+from multiprocessing import Process, Manager
+
+from flask import Flask
+from flask import request
+
+from ftxwebsocket.client import FtxWebsocketClient
+
+
+class TradeUpdateStreamer:
```

Comment:
Encapsulates logic from Part 1 to stream trades for a particular market, writing them to a multi-process dictionary that is snapshotted & used for index price computation when a REST API request is received. See https://github.com/73e255823b/ftx/pull/2/files#r873265771.